### PR TITLE
Remove dead code for corefx - System.Threading.Tasks.Dataflow

### PR DESCRIPTION
**Pull request by jiangzeng**
Remove dead code according to #17905 
Change for System.Threading.Tasks.Dataflow:
http://tempcoverage.blob.core.windows.net/report2/System.Threading.Tasks.Dataflow.diff.html

### CODE REVIEW

File: src/System.Threading.Tasks.Dataflow/src/System.Threading.Tasks.Dataflow.csproj [hunk 1]
```diff
@@ -52,9 +52,6 @@
     <Compile Include="Internal\TargetCore.cs" />
     <Compile Include="Internal\TargetRegistry.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.0' OR '$(TargetGroup)' == 'netstandard1.1'">
-    <Compile Include="Internal\Threading.cs" />
-  </ItemGroup>
```
Comment (stephentoub) [inline src/System.Threading.Tasks.Dataflow/src/System.Threading.Tasks.Dataflow.csproj]: Please revert this.  It's still necessary for one of the builds.

File: src/System.Threading.Tasks.Dataflow/src/Resources/Strings.resx [hunk 2]
```diff
@@ -94,7 +94,4 @@
   <data name="NotSupported_MemberNotNeeded" xml:space="preserve">
     <value>This member is not supported on this dataflow block. The block is intended for a specific purpose that does not utilize this member.</value>
   </data>
-  <data name="ConcurrentCollection_SyncRoot_NotSupported" xml:space="preserve">
-    <value>The SyncRoot property may not be used for the synchronization of concurrent collections.</value>
-  </data>
```
Comment (stephentoub) [inline src/System.Threading.Tasks.Dataflow/src/Resources/Strings.resx]: Please revert this.  It's still necessary for one of the builds.

File: src/System.Threading.Tasks.Dataflow/src/Internal/Threading.cs [hunk 3]
```diff
@@ -1,53 +0,0 @@
...
-        internal static void QueueUserWorkItem(WaitCallback callback, object state)
-        {
-            _ctx.Post(s =>
-            {
-                var tuple = (Tuple<WaitCallback, object>)s;
-                tuple.Item1(tuple.Item2);
-            }, Tuple.Create(callback, state));
-        }
-    }
-}
```
Comment (stephentoub) [inline src/System.Threading.Tasks.Dataflow/src/Internal/Threading.cs]: Please revert this.  It's still necessary for one of the builds.

Comment (danmosemsft) [inline src/System.Threading.Tasks.Dataflow/src/Internal/Threading.cs]: @jiangzeng stephen means revert the whole file.

**Comment by danmosemsft** [comment]
Looks good apart from those comments. Thanks!

**Comment by danmosemsft** [comment]
@jiangzeng pelase go ahead and start on another library for this task.

File: src/System.Threading.Tasks.Dataflow/src/Internal/EnumerableDebugView.cs [hunk 4]
```diff
@@ -35,23 +35,4 @@ public EnumerableDebugView(IEnumerable<KeyValuePair<TKey, TValue>> enumerable)
         [DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]
         public KeyValuePair<TKey, TValue>[] Items { get { return _enumerable.ToArray(); } }
     }
-
-    /// <summary>Debugger type proxy for an enumerable of T.</summary>
-    internal sealed class EnumerableDebugView<T>
```
Comment (alexperovich) [inline src/System.Threading.Tasks.Dataflow/src/Internal/EnumerableDebugView.cs]: Are we sure this is actually dead code? DebuggerTypeProxy can have a string parameter.

Comment (danmosemsft) [inline src/System.Threading.Tasks.Dataflow/src/Internal/EnumerableDebugView.cs]: Good catch. How is this still compiling?
```c#
    [DebuggerTypeProxy(typeof(EnumerableDebugView<,>))]
    internal sealed class QueuedMap<TKey, TValue>
```

@erozenfeld what am I missing here?

Comment (stephentoub) [inline src/System.Threading.Tasks.Dataflow/src/Internal/EnumerableDebugView.cs]: > Good catch. How is this still compiling?

That's a different type: `EnumerableDebugView<,>`, not `EnumerableDebugView<>`

Comment (jiangzeng) [inline src/System.Threading.Tasks.Dataflow/src/Internal/EnumerableDebugView.cs]: @stephentoub So do we still need to revert this code?

File: src/System.Threading.Tasks.Dataflow/src/Internal/Threading.cs (hunk 3 above)
Comment (jiangzeng) [inline src/System.Threading.Tasks.Dataflow/src/Internal/Threading.cs]: @danmosemsft Got it, will revert Threading.cs

File: src/System.Threading.Tasks.Dataflow/src/Internal/EnumerableDebugView.cs (hunk 4 above)
Comment (stephentoub) [inline src/System.Threading.Tasks.Dataflow/src/Internal/EnumerableDebugView.cs]: No, it's fine to remove this as you've done.

**Comment by jiangzeng** [comment]
@danmosemsft @stephentoub PR updated with your suggestions. Please check.

**Pull request merged**